### ISSUE TITLE
Allow multiple inputs

### ISSUE THREAD

**Issue by NickGeek** [pkra/MathJax-single-file#30]
It would be nice to be able to create a bundle, say with AsciiMath _and_ TeX inputs.

**Comment by pkra** [comment]
I'm open to a PR.

**Comment by Smibu** [comment]
Yeah, I've got the same problem. I tried to work this around by simply adding the AsciiMath files under `TeX` in the `input` variable in `main.js`, and building with `TeX`, but it seems like it isn't enough. The AsciiMath files do get bundled in the output file, but AsciiMath still isn't getting processed. Not getting any errors either. TeX input works fine.

To be clear, it looks like:

```
    TeX: [
        '/jax/input/TeX/config.js',
        '/jax/input/TeX/jax.js',
        '/jax/input/AsciiMath/config.js',
        '/jax/input/AsciiMath/jax.js'
    ],
```

**Comment by pkra** [comment]
@smibu that would be due to https://github.com/pkra/MathJax-single-file/issues/37. If you add `extensions/asciimath2jax.js` it should work.

Anyway, I'm still open for a PR to accept arrays of inputs. I just don't have time myself.
